packages/material-ui/src/useMediaQuery/useMediaQuery.test.js

```diff
@@ -51,142 +51,158 @@ describe('useMediaQuery', () => {
-      assert.strictEqual(values.callCount, 1);
+      mount(<Test />);
+      assert.strictEqual(text(), 'false');
+      assert.strictEqual(values.callCount, 2);
     });
   });
 
   it('should try to reconcile only the first time', done => {
+    const ref = React.createRef();
+    const text = () => ref.current.textContent;
     const Test = () => {
       const matches = useMediaQuery('(min-width:2000px)', {
         defaultMatches: true,
       });
       values(matches);
-      return <span>{`${matches}`}</span>;
+      return <span ref={ref}>{`${matches}`}</span>;
     };
 
-    let wrapper = mount(<Test />);
-    assert.strictEqual(wrapper.text(), 'false');
-    assert.strictEqual(values.callCount, 2);
+    mount(<Test />);
+    assert.strictEqual(text(), 'false');
+    assert.strictEqual(values.callCount, 4);
     setTimeout(() => {
       ReactDOM.unmountComponentAtNode(mount.attachTo);
-      wrapper = mount(<Test />);
-      assert.strictEqual(wrapper.text(), 'false');
-      assert.strictEqual(values.callCount, 3);
+      mount(<Test />);
+      assert.strictEqual(text(), 'false');
+      assert.strictEqual(values.callCount, 6);
 
       setTimeout(() => {
-        assert.strictEqual(wrapper.text(), 'false');
-        assert.strictEqual(values.callCount, 3);
+        assert.strictEqual(text(), 'false');
+        assert.strictEqual(values.callCount, 6);
         done();
       });
     });
   });
 
   it('should be able to change the query dynamically', () => {
+    const ref = React.createRef();
+    const text = () => ref.current.textContent;
     const Test = props => {
       const matches = useMediaQuery(props.query, {
         defaultMatches: true,
       });
       values(matches);
-      return <span>{`${matches}`}</span>;
+      return <span ref={ref}>{`${matches}`}</span>;
     };
     Test.propTypes = {
       query: PropTypes.string.isRequired,
     };
 
     const wrapper = mount(<Test query="(min-width:2000px)" />);
-    assert.strictEqual(wrapper.text(), 'false');
-    assert.strictEqual(values.callCount, 2);
-    wrapper.setProps({ query: '(min-width:100px)' });
-    assert.strictEqual(wrapper.text(), 'true');
+    assert.strictEqual(text(), 'false');
     assert.strictEqual(values.callCount, 4);
+    wrapper.setProps({ query: '(min-width:100px)' });
+    assert.strictEqual(text(), 'true');
+    assert.strictEqual(values.callCount, 8);
   });
 
   it('should observe the media query', () => {
+    const ref = React.createRef();
+    const text = () => ref.current.textContent;
     const Test = props => {
       const matches = useMediaQuery(props.query);
       values(matches);
-      return <span>{`${matches}`}</span>;
+      return <span ref={ref}>{`${matches}`}</span>;
     };
     Test.propTypes = {
       query: PropTypes.string.isRequired,
     };
 
-    const wrapper = mount(<Test query="(min-width:2000px)" />);
-    assert.strictEqual(values.callCount, 1);
-    assert.strictEqual(wrapper.text(), 'false');
```

Comment:
Probably some rebase issue.

Reply:
@oliviertassinari  I don't think I changed it, did I? Just switched from `ReactWrapper#text()` to `ref.current.textContent`. 

`ReactWrapper#text()` uses `findDOMNode()` which is why I'm using refs. Is more verbose but trade test confidence for verbosity anytime since I only have to write once.

Reply:
It's this switch I was wondering about. I didn't know .text() was using findDOMNode() internally.